vllm/model_executor/models/mpt.py

```diff
@@ -6,6 +6,7 @@
 import torch
```

Comment:
I'm not sure whether it's a good idea to add quantization for MPT in this PR. Because we will immediately refactor the code after this PR, adding quantization for MPT in this PR will make the refactor in the future harder.

Reply:
Sounds good. I'll just do Llama in this PR and we can add quantization to other models soon after.